src/core/WorkflowContext.ts

```diff
@@ -465,7 +465,7 @@ export class WorkFlowContext {
     protected writeSourceMaps(result: any) {
         // Writing sourcemaps
         if (this.sourceMapsProject || this.sourceMapsVendor) {
-            this.output.write(`${this.bundle.name}.js.map`, result.sourceMap);
+            this.output.write(`${this.output.filename}.js.map`, result.sourceMap);
```

Comment:
That's not correct. It has to be `bundle.name` 
It takes it from `fuse.bundle("vendor")` - `vendor` for example